79.md

```diff
@@ -0,0 +1,133 @@
+NIP-79
+======
+
+Digital Contracts
+-----------------
+
+`draft` `optional` `author:vpirato` `author:vitorpamplona`
+
+### Abstract
+
+This NIP proposes a new event type, `kind:900`, to represent digital contracts, covenants, and agreements within the Nostr protocol. These classifications are conceptual and aim to categorize the content of the contract. These can be signed, verified, and referenced by other events in the protocol.
+
+### Motivation
+
+As digital transactions, promises, and agreements become more prevalent, it's essential to have a standardized representation of contracts, covenants, and agreements in Nostr. This NIP aims to provide a structure and guidelines for creating, signing, and managing these within Nostr.
+
+### Specification
+
+1. **Types**:
+   - **Contract**: A legally binding agreement between two or more parties.
+   - **Covenant**: A written promise or restriction within a contract or agreement.
+   - **Agreement**: A mutual understanding between parties regarding their relative rights and responsibilities.
+
+2. **Format**: Following NIP-23 (Long-form Content), the content should be in Markdown format to structure and present the content in a human-readable manner.
+
+3. **Metadata**: Using NIP-78 (Application-specific data), specific application metadata fields can be added to the event, such as:

```

Comment:
No need to cite NIP-78. The list of tag here is also outdated from the examples below.

Reply:
what should i do? cancel this PR and make a new one with new commit? 🤔

Reply:
Just do a new commit, it should update the PR here